ra/ra.go

```diff
@@ -72,6 +74,93 @@ type RegistrationAuthorityImpl struct {
 	totalCertsStats      metrics.Scope
 }
 
+// Note: the issuanceExpvar must be a global. If it is a member of the RA, or
+// initialized with everything else in NewRegistrationAuthority() then multiple
+// invocations of the constructor (e.g from unit tests) will panic with a "Reuse
+// of exported var name:" error from the expvar package.
+var issuanceExpvar = expvar.NewInt("successfulIssuances")
+
+// Wrap time.Tick so we can override it in tests.
+var makeTicker = func(tickerDuration time.Duration) (func(), <-chan time.Time) {
+	t := time.NewTicker(tickerDuration)
+	return t.Stop, t.C
+}
+
+// The issuanceLoadStat allows us to expose a window of RA issuance volume as an
+// expvar. Internally a circular slice of 60 counts is maintained (one for each
+// minute). When a certificate is issued we increment the count in the current
+// minutes' bucket. A Go routine monitoring a ticket updates the expvar
+// periodically with the past 5 minutes of buckets totalled up.
+type issuanceLoadStat struct {
+	sync.RWMutex
+	issuedCounts []int
+	windowSize   int
+	ticker       <-chan time.Time
+	clk          clock.Clock
+}
+
+// Spawn a Go routine blocking on the ticker
+func (i *issuanceLoadStat) updateForever() {
+	go func() {
+		for {
+			// Compute the count and update the expvar immediately so the
+			// first round of the go-routine sets the initial expvar value
+			count := i.countIssuances()
+			issuanceExpvar.Set(int64(count))
+			// Then block on the ticker until its time to update again
+			<-i.ticker
```

Comment:
Did you consider a `time.Sleep` here? It might be simpler and we already have a mock Sleep in fakeclock.